tools/src/dbgen/README.md

```diff
@@ -9,37 +9,18 @@ fail with the "wrong magic" error.
 
 ## Usage
 
-First compile `dbgen` against the same revision of Cardano you want it to generate the `wallet-db` for.
-At the moment the program has been tested with `release/1.0.4`
-(SHA: `6e53bf599097aa0b55738a454115f76f69c9489e`). Then edit the `config.dhall` file to specify the number
-of wallets, accounts and addresses to generate. For example, the following config will generate 10 wallets,
-each wallet with 1 account and 100 addresses underneath:
-
+A typical usage is to run the tool from the branch/commit your wallet is on. 
+The properties file should look something like this:
 ```
 { wallet_spec = { account_spec = { addresses = 100 }, accounts = 1 }
-, wallets     = 10
+, fakeUtxo    = { fromAddress = 100, amount = 10000 }
+, wallets     = 1
 }
 ```
 
-Once you are ready, you can invoke `dbgen`:
-
+You should run the program using something like (from say `cardano-sl` root):
 ```
-Usage: dbgen [--config CONFIG.DHALL] [--dbPath rocksdb-path]
```

Comment:
My 2 cents, but I still think that's nice to show in the README the fact you can also use the tool as a poor man's benchmarking tool.